src/platform/silabs/wifi/SiWx/WifiInterfaceImpl.cpp

```diff
@@ -595,61 +592,32 @@ void WifiInterfaceImpl::ProcessEvent(WiseconnectWifiInterface::WifiPlatformEvent
         JoinWifiNetwork();
         break;
 
-    case WiseconnectWifiInterface::WifiPlatformEvent::kStationDoDhcp:
-        ChipLogDetail(DeviceLayer, "WifiPlatformEvent::kStationDoDhcp");
-        StartDHCPTimer(kDhcpPollIntervalMs);
```

Comment:
`kDhcpPollingIntervalMs` doesn't need to be in the WiseconnectWifiInterface anymore.